assets/css/responsive-utilities.less

```diff
@@ -22,7 +22,7 @@
     height: @menu-mobile-height;
     padding: 8px;
     [src*="opensuse-logo.png"] {
-      height: 27px;
+      height: 27px!important;
```

Comment:
Can we remove this? We shouldn't use !important since it makes the code harder to debug and overwrite when needed.
We can overwrite the class that changes the height for mobiles, that should be a better fix for this :)